File: swagger/models/unavailability.py
from .base_model import BaseModel


class Unavailability(BaseModel):
	"""
	no description available

	Attributes:

		id
		The ID of the unavailability.

		start_date_time
		The date and time the unavailability starts.

		end_date_time
		The date and time the unavailability ends.

		description
		A description of the unavailability.
	"""

	input_map = {
		'Id': 'id',
		'StartDateTime': 'start_date_time',
		'EndDateTime': 'end_date_time',
		'Description': 'description',
		}

	output_map = {
		'id': 'Id',
		'start_date_time': 'StartDateTime',
		'end_date_time': 'EndDateTime',
		'description': 'Description',
		}

	id = None
	start_date_time = None
	end_date_time = None
	description = None

File: swagger/Models/Unavailability.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class Unavailability extends BaseModel
{
	/**
	 * The ID of the unavailability.
	 * 
	 * @var integer
	 */
	public $id;
	/**
	 * The date and time the unavailability starts.
	 * 
	 * @var string
	 */
	public $startDateTime;
	/**
	 * The date and time the unavailability ends.
	 * 
	 * @var string
	 */
	public $endDateTime;
	/**
	 * A description of the unavailability.
	 * 
	 * @var string
	 */
	public $description;
}
